packages/cockpit/src/transactions/metrics/result.js

```diff
@@ -0,0 +1,77 @@
+import {
+  apply,
+  applySpec,
+  divide,
+  head,
+  is,
+  juxt,
+  map,
+  path,
+  pipe,
+  prop,
+  sortBy,
+  toString,
+  unless,
+} from 'ramda'
+
+const getBucketsPropFrom = propName => path([propName, 'buckets'])
+
+const formatBuckets = map(applySpec({
+  title: pipe(
+    prop('key'),
+    unless(
+      is(String),
+      toString
+    )
+  ),
+  value: prop('doc_count'),
+}))
+
+const getMetrics = pipe(
+  prop('aggregations'),
+  getBucketsPropFrom('metrics'),
+  head
+)
+
+const getVolume = pipe(
+  getMetrics,
+  path(['volume', 'value'])
+)
+
+const getTotalTransactions = path(['hits', 'total'])
+
+const getAverageAmount = pipe(
```

Comment:
acho que se usar um `converge` ao inves de pipe e juxt nesse caso fica um codigo mais facil de entender, o que acha?
```suggestion
const divideTruncate = pipe(
  divide, 
  Math.trunc
)
const getAverageAmount = converge(
  divideTruncate, 
  [getVolume, getTotalTransactions]
)
```

Reply:
Boa! Mudei para usar o `converge`